### DISCUSSION TITLE
Icon request: penguin

### DISCUSSION THREAD

**Discussion by si458** [FortAwesome/Font-Awesome#21204] [Icon Requests]
### What can it be used for?

it can be used for a zoo website along with other animal icons

### Example image (optional)

https://png.icons8.com/windows/1600/pinguin.png

### Icon request checklist

- [x] This is a single icon or matched pair (Ex: `lock` / `unlock`)
- [x] The title starts with "Icon request: " and is followed by the requested icon name (Ex: `Icon request: magical-unicorn`)
- [x] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate
- [x] I have included at least one example image if it could be helpful (optional)
- [ ] The request is for a concrete object (it's harder to make an icon to represent happiness, it's easier to make a smiley face. ☺)

**Comment by Crissov** [comment]
Unicode emoji #222:
🐧

**Comment by AutismPatient** [comment]
ws

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by jtheisen** [comment]
+1, I was just looking for a penguin icon for my attempt to bring font awesome in as symbols for observable plot, and they have a penguin default example data set.
